opsdroid/connector/slack/__init__.py

```diff
@@ -62,14 +70,14 @@ def __init__(self, config, opsdroid=None):
             ).data
             self.bot_id = self.user_info["user"]["profile"]["bot_id"]
 
-            _LOGGER.debug(_("Connected as %s"), self.bot_name)
-            _LOGGER.debug(_("Using icon %s"), self.icon_emoji)
-            _LOGGER.debug(_("Default room is %s"), self.default_target)
-            _LOGGER.info(_("Connected successfully"))
+            _LOGGER.debug(_("Connected as %s."), self.bot_name)
+            _LOGGER.debug(_("Using icon %s."), self.icon_emoji)
+            _LOGGER.debug(_("Default room is %s."), self.default_target)
+            _LOGGER.info(_("Connected successfully."))
         except slack.errors.SlackApiError as error:
             _LOGGER.error(
                 _(
-                    "Unable to connect to Slack due to %s - "
+                    "Unable to connect to Slack due to %s."
```

Comment:
This is another example that the two strings should be together like: "Unable to connect to Slack due to %s. The Slack Connector will not be available."

Reply:
Agreed.